static/bidder-info/mediasquare.yaml

```diff
@@ -0,0 +1,12 @@
+endpoint: "https://pbs-front.mediasquare.fr/msq_prebid"
```

Comment:
![image](https://github.com/user-attachments/assets/30cd4c2c-9d1f-47e5-9318-372a9581ffb6)
it works